apps/stopwatch/src/App.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { isTriggered } from "@felvin-search/core";
+
+//------------Styled Components-------------
+// If you're unfamiliar with styled components
+// start here https://styled-components.com/docs/basics#getting-started
+
+
+
+//=========================================
+
+// Your UI logic goes here.
+// `data` prop is exactly what is returned by queryToData.
+function Component({ data }) {
+
+  const [currentTime,setCurrentTime] = useState(0);
+  const [running,setRunning] = useState(false);
+  const [reset,setReset] = useState(false);
+
+  
+  useEffect(() => {
+    let interval;
+    if (running) {
+      interval = setInterval(() => {
+        setCurrentTime((prevTime) => prevTime + 10);
+      }, 10);
+    } 
+    else if(reset){
+      clearInterval(interval);
+      setReset(false);
+      setRunning(false);
+    }
+    return () => clearInterval(interval);
+  }, [running]);
+
+  const startStopButton = ()=>{
+    if(!running){
+      setRunning(true);
+    }
+    else{
+      setRunning(false);
+    }
+  }
+
+  const resetButton = ()=>{
+    if(running){
+      setRunning(false);
+      setCurrentTime(0);
+    }
+    else{
+      setCurrentTime(0);
+    }
+  }
+
+  return (
+    <div style={{justifyContent:"center", textAlign:"center",width:"100%"}}>
+      
+      <h1 >{(Math.floor(currentTime/3600000) === 0)?"":(Math.floor(currentTime/3600000))+"h "}{(Math.floor(currentTime/60000) === 0)&&(Math.floor(currentTime/3600000) === 0)?"":(("0"+(Math.floor(currentTime/60000))%60)).slice(-2)+"m"}{" "+("0"+((Math.floor(currentTime/1000))%60)).slice(-2)}s{" "+("0"+(Math.floor(currentTime/10))%100).slice(-2)}</h1>
+        
+      <p >
+        <button style={{backgroundColor:(running)?"red":"green",color:"white",padding:"10px",margin:"10px",fontSize:"large"}} onClick={startStopButton}>{running?"STOP":"START"}</button>
```

Comment:
We are using styled components for our project and hence it would be better if we don't use inline styling and convert these to styled components. You can look at other instant apps or styled components documentation to learn how they work.

Reply:
@sahil-shubham  mentor , I have removed the inline css and added styled components in my new commits.

Reply:
I really liked your idea of using different colours for start, stop and reset button. Can we implement the same with styled components? 

[This link](https://styled-components.com/docs/basics#adapting-based-on-props) would help in adapting the styling based on props in styled components.

Reply:
@sahil-shubham  mentor , in my new commits I have implemented those colors using styled components